app/src/main/java/chat/rocket/android/authentication/presentation/AuthenticationPresenter.kt

```diff
@@ -52,5 +53,9 @@ class AuthenticationPresenter @Inject constructor(
     fun privacyPolicy(toolbarTitle: String) =
         serverInteractor.get()?.let { navigator.toWebPage(it.privacyPolicyUrl(), toolbarTitle) }
 
+    fun saveDeepLinkInfo(deepLinkInfo: DeepLinkInfo) = navigator.saveDeepLinkInfo(deepLinkInfo)
```

Comment:
What about adding a blank line after each function declaration? It keeps the class more organized.